app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
   });
 };
 navSlide();
+
+// portifolio begin
+
+const portfolio = document.getElementById('recent-works');
+const projectsWrapperDiv = document.createElement('div');
+const mostRecentProjectWrapper = document.createElement('div');
+const descktopProjectsWrapperDiv = document.createElement('div');
+let projectDiv;
+
+const recentProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      " A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './images/Img Placeholder-p1.png',
+    workStack: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  },
```

Comment:
- [ ] 👍 Thank you for your efforts in the project. You have successfully used a JavaScript array to store information for each project. However, as per the project requirements, each object in the array should contain additional information, which is currently missing. To meet the project requirements, each project object in the array should have the following properties: name, description, featured image, technologies, link to the live version, and link to the project source code. 

Here's an example of how the project object array should be structured:

```javascript
const projects = [{
    name: "Project Title",
    description: "Project Description",
    image: "link_to_image.jpg",
    technologies: ["CSS", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
    source: "source link"
}]
```

Please make sure to add the missing information for each project in the array to align with the project requirements. Keep up the good work, and happy coding! 🌟🚀💻